velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,40 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add :doc:`json_extract_scalar <../functions/json>` function.
+* Update :doc:`Mathematical Functions <../functions/math>`.
+* Update :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide with new registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide with complex types.
+
+
+Core Library
+------------
+
+* Fix a bug in HashJoin filter to enable TPC-H query 19.
+* Add support for partition key aliases to Hive Connector.
+* Add support for conversions between Velox string and Arrow string.
+* Add support for 'timestamp with timezone'.
+* Add support for masks to StreamingAggregation operator.
```

Comment:
nit: perhaps, replace "masks" with "aggregation masks".

Actually, I'm seeing streaming aggregation itself being added in December. Therefore, let's change this to "Add StreamingAggregation operator". It would be nice to add a link to this section in the docs: https://facebookincubator.github.io/velox/develop/operators.html#aggregationnode-and-streamingaggregationnode

```
42c2088	Masha Basmanova	Mon Dec 20 13:19:15 2021 -0800	Add streaming aggregation (#771)
```